build.gradle.kts

```diff
@@ -32,6 +32,12 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
     implementation("org.springframework.boot:spring-boot-starter-security")
     implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.boot:spring-boot-starter-mail")
+    implementation("org.springframework.boot:spring-boot-starter-validation")
+    implementation("org.springframework.boot:spring-boot-starter-thymeleaf")
+    implementation("com.sun.mail:jakarta.mail:2.0.1")
+    implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.2.0")
```

Comment:
who asked?

Reply:
It show 4 line dependencies, I don't know what to remove

Reply:
org.springdoc:springdoc-openapi-starter-webmvc-ui:2.2.0

Reply:
implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.2.0")
 redundant web mvc

Reply:
@TranSiTien remove 2.2.0 instead 2.6.0